x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/RegressionIT.java

```diff
@@ -544,19 +543,26 @@ public void testAliasFields() throws Exception {
         startAnalytics(jobId);
         waitUntilAnalyticsIsStopped(jobId);
 
+        double predictionErrorSum = 0.0;
+
         SearchResponse sourceData = client().prepareSearch(sourceIndex).setSize(totalDocCount).get();
         for (SearchHit hit : sourceData.getHits()) {
             Map<String, Object> destDoc = getDestDoc(config, hit);
             Map<String, Object> resultsObject = getMlResultsObjectFromDestDoc(destDoc);
 
-            int featureValue = (int) destDoc.get("field_1");
-            double predictionValue = (double) resultsObject.get(predictionField);
-            assertThat(predictionValue, closeTo(2 * featureValue, 10.0));
-
             assertThat(resultsObject.containsKey(predictionField), is(true));
             assertThat(resultsObject.containsKey("is_training"), is(true));
+
+            int featureValue = (int) destDoc.get("field_1");
+            double predictionValue = (double) resultsObject.get(predictionField);
+            predictionErrorSum += Math.abs(predictionValue - 2 * featureValue);
```

Comment:
I'm wondering if we could drop the `Math.abs` here. This would of course increase the chance of the assertion in line 564 to pass but maybe it is not necessary.

Reply:
We are interested in how far the prediction is from what we expect. Dropping the `abs` would mean that 2 errors that are opposite direction would cancel each other out and possibly this could hide the fact that the error is too large.